### PR TITLE
Update outdated in Korean resource-quotas.md

### PR DESCRIPTION
**Pull request by seokho-son**
This PR is to update outdated file in dev-1.19-ko.5 (part-01). Issue: #24860

This PR contains the following updated file
content/ko/docs/concepts/policy/resource-quotas.md
(for content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 102 insertions(+), 89 deletions(-))

/language ko

### CODE REVIEW

**Comment by seokho-son** [comment]
@gochist 해당 PR의 리뷰를 부탁 드립니다.. ^^
/assign @gochist

**Comment by seokho-son** [comment]
https://deploy-preview-25261--k8s-dev-ko.netlify.app/ko/docs/concepts/policy/resource-quotas/

File: content/ko/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -70,10 +70,14 @@ API 서버 `--enable-admission-plugins=` 플래그의 인수 중 하나로
...
-| `limits.cpu` | 터미널이 아닌 상태의 모든 파드에서 CPU 제한의 합은 이 값을 초과할 수 없음 |
-| `limits.memory` | 터미널이 아닌 상태의 모든 파드에서 메모리 제한의 합은 이 값을 초과할 수 없음 |
-| `requests.cpu` | 터미널이 아닌 상태의 모든 파드에서 CPU 요청의 합은 이 값을 초과할 수 없음 |
-| `requests.memory` | 터미널이 아닌 상태의 모든 파드에서 메모리 요청의 합은 이 값을 초과할 수 없음 |
+| `limits.cpu` | 터미널이 아닌 상태의 모든 파드에서 CPU 제한의 합은 이 값을 초과할 수 없음. |
+| `limits.memory` | 터미널이 아닌 상태의 모든 파드에서 메모리 제한의 합은 이 값을 초과할 수 없음. |
+| `requests.cpu` | 터미널이 아닌 상태의 모든 파드에서 CPU 요청의 합은 이 값을 초과할 수 없음. |
+| `requests.memory` | 터미널이 아닌 상태의 모든 파드에서 메모리 요청의 합은 이 값을 초과할 수 없음. |
+| `hugepages-<size>` | 터미널 상태가 아닌 모든 파드에 걸쳐서, 지정된 사이즈의
+휴즈 페이지 요청은 이 값을 초과하지 못함. |
```
Comment (gochist) [inline content/ko/docs/concepts/policy/resource-quotas.md:78]: 테이블 렌더링이 깨지는데, 원문도 동일하네요.

**Comment by gochist** [comment]
@seokho-son 감사합니다!
/lgtm
/approve

**Pull request merged**